src/gui/optionsdialog.ui

```diff
@@ -3131,6 +3131,81 @@ Disable encryption: Only connect to peers without protocol encryption</string>
+             <widget class="QGroupBox" name="checkAddTrackersFromURL">
+              <property name="title">
+               <string>Automatically append trackers from URL to new downloads:</string>
+              </property>
+              <property name="checkable">
+               <bool>true</bool>
+              </property>
+              <property name="checked">
+               <bool>false</bool>
+              </property>
+              <layout class="QGridLayout" name="gridLayout_999">
+               <property name="leftMargin">
+                <number>9</number>
+               </property>
+               <property name="topMargin">
+                <number>9</number>
+               </property>
+               <property name="rightMargin">
+                <number>9</number>
+               </property>
+               <property name="bottomMargin">
+                <number>9</number>
+               </property>
+               <item row="0" column="0">
+                <widget class="QLabel" name="labelCustomizeTrackersListUrl">
+                 <property name="text">
+                  <string>URL:</string>
+                 </property>
+                </widget>
+               </item>
+               <item row="0" column="1">
+                <widget class="QLineEdit" name="textTrackersURL">
+                 <property name="enabled">
+                  <bool>true</bool>
+                 </property>
+                 <property name="text">
+                  <string>https://example.com</string>
+                 </property>
+                 <property name="readOnly">
+                  <bool>false</bool>
+                 </property>
+                 <property name="cursorMoveStyle">
+                  <enum>Qt::LogicalMoveStyle</enum>
+                 </property>
+                </widget>
+               </item>
+               <item row="1" column="0" colspan="2">
+                <widget class="QGroupBox" name="trackersFromURLBox">
+                 <property name="title">
+                  <string>Fetched trackers</string>
+                 </property>
+                 <layout class="QGridLayout" name="gridLayout_112">
+                  <item row="1" column="0" colspan="2">
+                   <widget class="QPlainTextEdit" name="textTrackersFromURL">
+                    <property name="enabled">
+                     <bool>true</bool>
+                    </property>
+                    <property name="minimumSize">
+                     <size>
+                      <width>0</width>
+                      <height>153</height>
+                     </size>
+                    </property>
+                    <property name="readOnly">
+                     <bool>true</bool>
+                    </property>
+                   </widget>
+                  </item>
+                 </layout>
+                </widget>
+               </item>
+              </layout>
+             </widget>
+            </item>
```

Comment:
You have done it excessively, too many unrelated properties. I'll suggest a cleaner version:
```suggestion
            <item>
             <widget class="QGroupBox" name="checkAddTrackersFromURL">
              <property name="title">
               <string>Automatically append trackers from URL to new downloads:</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <layout class="QVBoxLayout" name="checkAddTrackersFromURLLayout">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_21">
                 <item>
                  <widget class="QLabel" name="labelCustomizeTrackersListUrl">
                   <property name="text">
                    <string>URL:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="textTrackersURL"/>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QLabel" name="labeltrackersFromURL">
                 <property name="text">
                  <string>Fetched trackers</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPlainTextEdit" name="textTrackersFromURL">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
```

Reply:
Tested and confirmed this is cleaner. Thank you